### ISSUE TITLE
batch spatial search

### ISSUE THREAD

**Issue by burritojustice** [heremaps/here-cli#300]
Allow multiple feature IDs for spatial searches:

    here xyz show dataSpace --spatial --feature adminSpace,90210,90125,20252

(If sent to a --targetSpace, these would run in parallel.)

Also allow iteration through a space of features and use the ids to do a spatial search:

    here xyz show dataSpace --spatial --feature IWxOWKeo,*

(or maybe no list of features, and just the space ID?)

We should also allow `--targetSpace` (currently only available in the h3 search) to be used to direct all spatial search output to a target space. (We should also automatically create a space if one is not specified.)

Stretch goal: use property search to select features from a space, and use matching features for a spatial search:

    here xyz show dataSpace --spatial --feature adminSpace,p.feature>10,p.feature2=x
